components/KitImagePicker.js

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import { Image, StyleSheet, View } from "react-native";
+
+import * as ImagePicker from "expo-image-picker";
+import * as Permissions from "expo-permissions";
+import KitText from "./KitText";
+import KitButtonSupreme from "./KitButtonSupreme";
+import Colors from "../constants/Colors";
+import FontStyles from "../constants/FontStyles";
+import { uploadImageAsync } from "../utils/firebase/storage";
+
+export default class KitImagePicker extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      image: null,
+      isPhotoSelected: false
+    };
+  }
+
+  async componentDidMount() {
+    await Permissions.askAsync(Permissions.CAMERA_ROLL);
```

Comment:
Is this a similar implementation to what we use in `ImageScreen.js` (i.e. the screen that is presented to the user when they're submitting a photo challenge?)

Reply:
If so we can consider componentizing the Image selector on `ImageScreen.js`

Reply:
yeah that was supposed to be what KitImagePicker was but i didn't change the ImageScreen to use KitImagePicker and tbh we can just not. maybe future image pickers can use this componenet